R/Connection.R

```diff
@@ -219,7 +219,8 @@ setMethod(
   "dbExistsTable", c("OdbcConnection", "character"),
   function(conn, name, ...) {
     stopifnot(length(name) == 1)
-    name %in% dbListTables(conn, ...)
+    df <- connection_sql_tables(conn@ptr, table_name = name)
+    nrow(df) > 0
```

Comment:
Could we use `NROW(df)` here just to be sure we will get a numeric value?

I am pretty sure that `connection_sql_tables()` will return an empty data.frame if there are no results, but using `NROW()` will still work if for some reason it instead returns `NULL`.

Reply:
I've made the change.